lib/services/local/access_list.go

```diff
@@ -31,21 +31,24 @@ import (
 )
 
 const (
-	accessListPrefix       = "access_list"
-	accessListMemberPrefix = "access_list_member"
-	accessListMaxPageSize  = 100
+	accessListPrefix      = "access_list"
+	accessListMaxPageSize = 100
+
+	accessListMemberPrefix      = "access_list_member"
+	accessListMemberMaxPageSize = 200
 )
 
 // AccessListService manages Access List resources in the Backend.
 type AccessListService struct {
-	log   logrus.FieldLogger
-	clock clockwork.Clock
-	svc   *generic.Service[*accesslist.AccessList]
+	log    logrus.FieldLogger
+	clock  clockwork.Clock
+	alSvc  *generic.Service[*accesslist.AccessList]
+	almSvc *generic.Service[*accesslist.AccessListMember]
```

Comment:
Ditch the unusual abbreviations? The larger the scope, the clearer the name should be.

```suggestion
	service  *generic.Service[*accesslist.AccessList]
	memberService *generic.Service[*accesslist.AccessListMember]
```